lib/rohd_hcl.dart

```diff
@@ -12,6 +12,7 @@ export 'src/error_checking/error_checking.dart';
 export 'src/exceptions.dart';
 export 'src/fifo.dart';
 export 'src/find.dart';
+export 'src/float_fixed.dart';
```

Comment:
Do you want to move this to arithmetic.dart?

Reply:
Done.